FindYourFriends/screens/MapScreen.js

```diff
@@ -12,8 +12,47 @@ import Card from "../components/Card";
 import colors from "../constants/colors";
 import Input from "../components/Input";
 import facade from "../facade";
+import useMutation from "@apollo/react-hooks";
 
 const MapScreen = (props) => {
+
+  const [updatePosition, { loading, error, data, called }] = useMutation(
+    facade.UPDATE_POSITION
+  );
+
+  // Example of how to use the hook
+  async function updateMyPosition(){
+    await updatePosition({
+      variables: {
+        username: "name",
+        coordinates: {
+          lon: 12.57,
+          lat: 55.66
```

Comment:
```suggestion
          lon,
          lat
```

Reply:
I may be **Mr. Hardcode** but I'm not *that* set on hardcoding things :P